docs/helm-chart-checks.md

```diff
@@ -131,18 +131,34 @@ This section provides help on the basic usage of Helm chart checks with the podm
 
 ### Cluster Config
 
-Configuration of the `chart-testing` check can be achieved through the `--set` command line option, which in addition of configuring `chart-verifier` itself, can set check's configuration values.
+Configuration of the `chart-testing` check can be achieved in one of two ways: 
+
+1. Through the `--set` command line option, which in addition of configuring `chart-verifier` itself, can set check's configuration values.
```

Comment:
Suggestion: Place this under the 1st step as the 1st option (as a bullet item).

Reply:
Done

Reply:
Thanks!